From 7a4ab2ac5886cad13999f61bbb9638f4b84dafc2 Mon Sep 17 00:00:00 2001
From: chenzhen <chenzhen@rock-chips.com>
Date: Tue, 4 Jul 2017 11:06:25 +0800
Subject: [PATCH] arm64: dts: rockchip: change "power-mode" to "power_model" in
 gpu node for rk3328

The reference code from ARM uses "power_model".
It's convenient to use the same name
for rebasing Mali DDK update from ARM and avoiding confusion.

Change-Id: I9095500a5d0eb7f9edfeed3a07a1c2bdca51aaf1
Signed-off-by: chenzhen <chenzhen@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3328.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3328.dtsi b/arch/arm64/boot/dts/rockchip/rk3328.dtsi
index df5495b6004a..f8f0dffe59ad 100644
--- a/arch/arm64/boot/dts/rockchip/rk3328.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3328.dtsi
@@ -600,7 +600,7 @@
 		operating-points-v2 = <&gpu_opp_table>;
 		status = "disabled";
 
-		gpu_power_model: power-model {
+		gpu_power_model: power_model {
 			compatible = "arm,mali-simple-power-model";
 			voltage = <900>;
 			frequency = <500>;
-- 
2.35.3

